### PR TITLE
4.7.3 - v4 Migration Cleanup

### PR DESCRIPTION
**Pull request by rejahrehim**
This PR part of issue #83 

- [ ] This PR handles the issue and requires no additional PRs.
- [x] You have validated the need for this change.

**What did this PR accomplish?**

- Fix linting

### CODE REVIEW

File: document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.3_Testing_for_Session_Fixation_OTG-SESS-003.md [hunk 1]
```diff
@@ -1,75 +1,83 @@
...
+# Testing for Session Fixation
+
+## Summary
 
 When an application does not renew its session cookie(s) after a successful user authentication, it could be possible to find a session fixation vulnerability and force a user to utilize a cookie known by the attacker. In that case, an attacker could steal the user session (session hijacking).
 
-Session fixation vulnerabilities occur when:\
-\* A web application authenticates a user without first invalidating the existing session ID, thereby continuing to use the session ID already associated with the user.
+Session fixation vulnerabilities occur when:
+
```
Comment (ThunderSon) [inline document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.3_Testing_for_Session_Fixation_OTG-SESS-003.md]: No need for the new line as this is a list

File: document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.3_Testing_for_Session_Fixation_OTG-SESS-003.md [hunk 2]
```diff
@@ -1,75 +1,83 @@
...
 Furthermore, the issue described above is problematic for sites that issue a session identifier over HTTP and then redirect the user to a HTTPS log in form. If the session identifier is not reissued upon authentication, the attacker can eavesdrop and steal the identifier and then use it to hijack the session.
 
-How to Test
------------
+## How to Test
 
 ### Black Box Testing
 
-**Testing for Session Fixation vulnerabilities:**\
+#### Testing for Session Fixation Vulnerabilities:
```
Comment (ThunderSon) [inline document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.3_Testing_for_Session_Fixation_OTG-SESS-003.md]: Remove the trailing column

File: document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.3_Testing_for_Session_Fixation_OTG-SESS-003.md [hunk 3]
```diff
@@ -81,20 +89,15 @@ Talk with developers and understand if they have implemented a session token ren
 
 **Result Expected:** The application should always first invalidate the existing session ID before authenticating a user, and if the authentication is successful, provide another sessionID.
```
Comment (ThunderSon) [inline document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.3_Testing_for_Session_Fixation_OTG-SESS-003.md]: What do you think about replacing `**result expected**` with `>`? For both lines 83 and 89. Waiting for your opinion 😄

File: document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.3_Testing_for_Session_Fixation_OTG-SESS-003.md [hunk 4]
```diff
@@ -1,75 +1,83 @@
...
-How to Test
------------
+## How to Test
 
 ### Black Box Testing
 
-**Testing for Session Fixation vulnerabilities:**\
+#### Testing for Session Fixation Vulnerabilities:
+
 The first step is to make a request to the site to be tested (example www.example.com). If the tester requests the following:
```
Comment (ThunderSon) [inline document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.3_Testing_for_Session_Fixation_OTG-SESS-003.md]: replace `(example www.example.com)` with 
```
(_e.g._ `www.example.com`)
```

File: document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.3_Testing_for_Session_Fixation_OTG-SESS-003.md (hunk 3 above)
Comment (rejahrehim) [reply document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.3_Testing_for_Session_Fixation_OTG-SESS-003.md]: So there will be only `>` in place of the Bold header?

File: document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.3_Testing_for_Session_Fixation_OTG-SESS-003.md (hunk 4 above)
Comment (rejahrehim) [reply document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.3_Testing_for_Session_Fixation_OTG-SESS-003.md]: @ThunderSon Updated with fix

File: document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.3_Testing_for_Session_Fixation_OTG-SESS-003.md (hunk 3 above)
Comment (rejahrehim) [reply document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.3_Testing_for_Session_Fixation_OTG-SESS-003.md]: Done

File: document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.3_Testing_for_Session_Fixation_OTG-SESS-003.md (hunk 1 above)
Comment (kingthorin) [reply document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.3_Testing_for_Session_Fixation_OTG-SESS-003.md]: This item is still oustanding.

Comment (rejahrehim) [reply document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.3_Testing_for_Session_Fixation_OTG-SESS-003.md]: I think there should be a blank line before and after lists ( `Lists should be surrounded by blank lines - markdown lint(MD032)`). I have removed all blank lines in between list entries. Please confirm @kingthorin

Comment (kingthorin) [reply document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.3_Testing_for_Session_Fixation_OTG-SESS-003.md]: Makes sense to me I'll leave the final call to @ThunderSon as it was his comment originally.

Comment (ThunderSon) [reply document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.3_Testing_for_Session_Fixation_OTG-SESS-003.md]: Correct

**Pull request merged**